src/components/shared/TheEntries.vue

```diff
@@ -6,7 +6,7 @@
   >
     <div class="flex items-center">
       <h2 class="q-my-auto text-bold text-h5 q-pa-md">Entries</h2>
-      <q-btn v-if="!hasWinner" color="primary" dense icon="add" outline round @click="openEntryDialog">
+      <q-btn v-if="hasWinner" color="primary" dense icon="add" outline round @click="openEntryDialog">
```

Comment:
Hello Pradnya! Thank you for the contribution 🙏
But:
1. It was fixed a little bit earlier in #466 by @sana7044 
2. As a review - this check is aimed to see if the prompt has a winning entry selected, the "Add new entry" button should be hidden.